.github/workflows/ci.yml

```diff
@@ -24,15 +24,22 @@ env:
   BUILDIFIER_SHA256SUM: '4c985c883eafdde9c0e8cf3c8595b8bfdf32e77571c369bf8ddae83b042028d6'
 
 jobs:
-  lint-python:
+  lint-flake8:
```

Comment:
nit: maybe call this `lint-python-flake8` just to be clear to the uninitiated that it's also linting python?  Since otherwise separating it out from `lint-python` is a little confusing.

Reply:
Sure; fine with me. Done.